### DISCUSSION TITLE
VENTOY UPDATER

### DISCUSSION THREAD

**Discussion by fidodido48** [ventoy/Ventoy#1911] [Q&A]
Hi,
does Ventoy have an update option to update itself from a github repo?
Such a basic option missing in 2022...

**Comment by ruftas** [comment]
It would be a great thing to have.

**Reply by RishonDev** [reply]
I wish I could do it if I knew C....

**Comment by uuaws2** [comment]
Nop , we dont have it neither on the tool itself or in the github project posts. Its kinda of frustirating.
